config/v201/config.json

```diff
@@ -776,10 +775,16 @@
                     "Actual": ""
                 }
             },
+            "SmartChargingCtrlrAvailable": {
```

Comment:
It looks like the word `Available` in the `SmartChargingCtrlrAvailableEnabled` variable is a typo from an older contribution should be removed. For this requirement, we should be using `Available`, rather than `Enabled` and will push an update fixing this (and removing the old typo).

I share your dislike and confusion around the Available and Enabled variable names, but they are unfortunately standard variables in the OCPP 2.0.1 device model that are defined for each controller component.

There is an ongoing discussion here that gets into more detail about when and where to use one versus the other:
https://lfenergy.zulipchat.com/#narrow/stream/417676-EVerest.3A-Cloud-communication/topic/Proper.20Use.20of.20Accepted.20and.20Enabled.20Controller.20Variables/near/460109230

Reply:
We will need to move this comment to an action item for later. While attempting to update `SmartChargingCtrlrAvailableEnabled` we dug into how the schema are generated and ran into some issue captured [here](https://github.com/EVerest/libocpp/issues/752#issue-2475955847)

Reply:
You can just rename that variable manually in the `component_config/SmartChargingCtrlr.json` and in `ctrlr_component_variables.hpp` and `ctrlr_component_variables.cpp`.

Reply:
Doesn't that risk being overwritten?

Reply:
Ok, addressed.